classes/Commands/CreateBackupCommand.php

```diff
@@ -64,9 +63,8 @@ protected function execute(InputInterface $input, OutputInterface $output): ?int
 
             return $exitCode;
         } catch (Exception $e) {
-            $this->logger->error('An error occurred during the backup process: ' . $e->getMessage());
-
-            return ExitCode::FAIL;
+            $this->logger->error('An error occurred during the  backup creation process');
```

Comment:
```suggestion
            $this->logger->error('An error occurred during the backup creation process');
```

One redundant space is found

Reply:
outdated